tools/mydumper.md

```diff
@@ -0,0 +1,46 @@
+---
+title: mydumper Instructions 
+summary: Use mydumper to export data from TiDB.
+category: tools
+---
+
+# mydumper Instructions
+
+## What is mydumper?
+
+`mydumper` is a fork of the [mydumper](https://github.com/maxbube/mydumper) project with additional functionality specific to TiDB. It is the recommended method to use for logical backups of TiDB.
+
+[Download the Binary](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz).
+
+## What enhancements does this contain over regular mydumper?
+
++ Use `tidb_snapshot` to provide backup consistency instead of `FLUSH TABLES WITH READ LOCK`
+
++ Include the hidden `_tidb_rowid` column in `INSERT` statements when present
```

Comment:
I assume the INSERT statements would include the column names? I get a "Column count doesn't match value count" error otherwise. 

```sql
INSERT INTO `table` (_tidb_rowid, col1, col2) VALUES (1, 'x', 'y'), ...;
--                  ^~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
The relevant line is:
https://github.com/pingcap/mydumper/blob/master/mydumper.c#L3035

If it knows there is a generated col or hidden rowid, it must use the complete insert syntax that specifies column names.  See testcase:

https://gist.github.com/morgo/38b7562fcea4f9839433afd7503d8099

Reply:
I see, thanks :)